slide-in-with-cta/template.css

```
@@ -0,0 +1,204 @@
+/* ==========================================================================
+   Slide-In
+========================================================================== */
+
+#evg-slide-in-with-cta {
+    position: absolute;
+    z-index: 100000;
+    top: auto !important;
+    right: 10%;
+    bottom: 10vh !important;
+    left: auto;
+    width: 300px;
+    max-width: 90vw;
+    height: 300px;
+    max-height: 90vh;
+    border: 1px solid #000;
+    color: #fff;
+    background-color: #000;
+    font-family: "Helvetica Neue", "Arial", sans-serif;
```

```suggestion
    font-family: inherit;
```
We don't want to assign font-family. 
See examples in other templates: https://github.com/evergage/evergage-global-templates/search?q=font-family